core/unix/inc/TUnixSystem.h

```diff
@@ -71,6 +78,17 @@ class TUnixSystem : public TSystem {
    static int          UnixRecv(int sock, void *buf, int len, int flag);
    static int          UnixSend(int sock, const void *buf, int len, int flag);
 
+   // added helper static members for stacktrace
+   static const int                    fPidStringLength = 255;
+   static char                         fPidString[fPidStringLength];
+   static int                          fParentToChild[2];
+   static int                          fChildToParent[2];
+   static std::unique_ptr<std::thread> fHelperThread;   
+   static char *const                  kStackArgv[];
+   static char *const *                GetStackArgv();
```

Comment:
This is not part of standard.  The advantage would to more clearly segregate the functionality (even some of the function could be assign to that struct rather than TSystem itself).  This would actually make re-use of the code in a different context (a new version of TSystem for example) easier.